### ISSUE TITLE
Fav v2 - Dimming request

### ISSUE THREAD

**Issue by meluvalli** [NorthernMan54/homebridge-alexa#478]
Was wondering if it was possible to support dimming function for fans.  
All my fans are on dimmer switches.  1-33% = Speed 1, 34-63% is Speed 2, and 64-100% = Speed 3.

I know Alexa supports dimming on fans, so not sure if you can just change something so it shows the dimming function in Alexa or not.

HomeBridge already sees it as dimmable.

**Comment by NorthernMan54** [comment]
Have you asked Alexa to set the fan to 33 or 50 or 75 ?

**Comment by meluvalli** [comment]
Opps!  No, I didn't try it!  I just looked at the device in Alexa app, and it doesn't show it's dimmable.  Actually, the app doesn't even let you turn it on or off like a light switch.  I wonder why this would be?  I don't have an actual "alexa compatable" fan, but  I looked at youtube videos and when they pull the switch up, they can dim it just like a light!

Voce command, saying "Alexa Office Fan 33%" and "Alexa Office Fan 99%" works fine.  So, just an issue with it being displayed wrong in the Alexa app!
